.gitignore

```diff
@@ -1,3 +1,4 @@
 .env
 .env.local
 .env.development
+/target
```

Comment:
How about having this be specific to each related rust project?

Reply:
That would not work. The target directory is located at `/target`. If I do what you're describing, it will only match `/src/rust/**/target`, which will not match `/target`.

Reply:
Per https://github.com/econia-labs/emojicoin-dot-fun/pull/25#pullrequestreview-2025533633, take out top-level gitignore since won't be needed